src/App.jsx

```diff
@@ -1,12 +1,15 @@
 import Header from "./Components/Header/Header";
 import Gallery from "./Components/Gallery/Gallery";
+import UploadModal from "./Components/UploadModal/UploadModal";
+import UploadButton from "./Components/UploadButton/UploadButton";
```

Comment:
This seems to be a component from another ticket. What's the reason for having this here?

Reply:
Added those to App.js to render in localhost while working on functionality and pre-deploy styling. Mistake on my part to leave those there. Feel free to remove them before merging.